crates/turbo-updater/src/lib.rs

```diff
@@ -61,7 +61,12 @@ pub fn check_for_updates(
     interval: Option<Duration>,
 ) -> Result<(), UpdateNotifierError> {
     // bail early if the user has disabled update notifications
-    if std::env::var(NO_UPDATE_NOTIFIER).is_ok() {
+    let use_update_notifier = match std::env::var(TURBO_NO_UPDATE_NOTIFIER) {
+        Ok(v) => v == "1",
+        Err(_) => false,
+    };
```

Comment:
I want us to constrain what the values are that we accept for this. I don't want to have to support "set empty string" for example because JS is a jerk about that.